README.rst

```diff
@@ -1,6 +1,9 @@
 MailDump
 ========
 
+.. image:: https://badge.fury.io/py/maildump.svg
+    :target: https://badge.fury.io/py/maildump
```

Comment:
I don't really like the idea of routing this through a third party... What about just using these URLs directly?

https://img.shields.io/pypi/v/maildump.svg
https://pypi.python.org/pypi/maildump/

I think shields.io is very similar to fury.io but I'm already using it in other projects so I'd prefer that.

About the placement, what about including it in the "MailDump" h1? IMO that placement looks [pretty nice](https://github.com/indico/indico/#indico----)

Reply:
They are using .md you are using .rst. I wanted to add the logo but rst kills README creativity

Reply:
ah crap, feel free to convert the readme to markdown - nowadays pypi lets you specify that a file is gfm markdown anyway via setup.cfg:

```
long_description = file: README.md
long_description_content_type = text/markdown; charset=UTF-8; variant=GFM
```

Reply:
Ok wanted to avoid cloning the project on my PC, there i go XD

Reply:
don't be lazy! :P